### DISCUSSION TITLE
How do we deactivate a team member

### DISCUSSION THREAD

**Discussion by Kishub** [madrecha/jamku#655] [QA] [answered]
Team member is no longer associated with firm anymore. How to remove him from jamku login.
Questions
1. Delete Team Member from Jamku
2. Deactivate Login
3. Block team member to Login (when on extended/exam leave

### Related Questions & Answers

1. [View List of Deactivated Team Members](https://github.com/madrecha/jamku/discussions/969)

**Comment by Kishub** [comment, marked as answer]
To deactivate a team member:
1. Go to team page from navigation bar
  ![image](https://user-images.githubusercontent.com/11911938/162428840-2e6f56c7-6359-4a6c-8b0e-426984aa7f3a.png)

1. Click on team member name whom you want to deactivate.
  ![image](https://user-images.githubusercontent.com/11911938/162431219-28c92df2-bd29-49aa-bc6a-fd2be8a3f5f6.png)

1. On the team view page, go to "Permissions" Tab
  ![image](https://user-images.githubusercontent.com/11911938/162430290-633366b1-6997-45b8-8d70-fb13f264f3d1.png)

1. Click on `Deactivate` button.
  ![image](https://user-images.githubusercontent.com/11911938/162431405-9b4a8cf9-fcc5-4648-9ca1-c7d0ef6715b5.png)

1. Now it will ask you for some questions to before actually deactivating team member. You can skip these questions, by clicking `Continue` button. Finally click on `Deactivate` Button.
  ![image](https://user-images.githubusercontent.com/11911938/162432014-73f892e0-9344-48ab-bc70-1b7a62d0d503.png)

The team member is now deactivated. 
However, all the data is safe. His workdone (aka Timesheet), Task Notes, etc will still be stored in Jamku. Nothing is deleted.
[View List of Deactivated Team Members](https://github.com/madrecha/jamku/discussions/969)


Note:
You will need to have "Owner" Permission to deactivate/activate team member.

**Comment by adarshmadrecha** [comment]
Practical Tips:
1. When a team member is going for an extended leave, deactivate the team member and again activate him when he joins back. This is recommended only if team member is on leave for more than one week.
  In this case, don't un-assign him from existing tasks (when deactivating). Simply click on continue and deactivate.

1. When a team member leaves the organisation, re-distribute his current tasks (i.e. tasks with Going and Stuck Status) to other team members. During deactivation, remove the team member from all the all the current tasks.
   ![image](https://user-images.githubusercontent.com/11911938/162435222-7ea105c1-035b-40b9-b4a7-6bb12a84bf64.png)
  and also remove the team member from all the other team member's Team.
  ![image](https://user-images.githubusercontent.com/11911938/162435576-62f02846-4ea7-43c7-b784-25dd6533f8b1.png)
